sample_apps/run_example.sh

```diff
@@ -8,27 +8,19 @@ set -Eeuo pipefail
 # Setup script globals, to establish curdir and root path to Certifier code base
 Me=$(basename "$0")
 
-
-# Establish full dirpath where this script lives
+SampleAppName="simple_app"
 pushd "$(dirname "$0")" > /dev/null 2>&1
-EXAMPLE_DIR=$(pwd); export EXAMPLE_DIR
+Cleanup="$(pwd)/cleanup.sh"
 popd > /dev/null 2>&1
 
-Cleanup="${EXAMPLE_DIR}/cleanup.sh"
-PROV_DIR="${EXAMPLE_DIR}/provisioning"
-
-# shellcheck disable=SC2046
-CERT_PROTO="$(dirname $(dirname "$EXAMPLE_DIR"))"; export CERT_PROTO
-CERT_UTILS="${CERT_PROTO}/utilities"
-ExName=$(basename "$EXAMPLE_DIR")
```

Comment:
This block moves down and these env-vars / scripts-symbols will be established based on the `sample-app-name` parsed as 1st arg.